### ISSUE TITLE
Time for a new release?

### ISSUE THREAD

**Issue by TastyPi** [rubocop/rubocop-thread_safety#21]
```ruby
class Foo
  def self.bar
    define_method :baz do
      @x = 10
    end
  end
end
```

```
foo.rb:4:7: C: ThreadSafety/InstanceVariableInClassMethod: Avoid instance variables in class methods.
      @x = 10
      ^^
```

In the context of `define_method` instance variables are actual instance variables, not class instance variables.

**Comment by koic** [comment]
@mikegee Can you cut a new release? Or you will grant me release permission to rubygems.org and I can do it.

**Comment by mikegee** [comment]
Oops. I don't keep up with my personal inbox during the week. Sorry. 

I intend to both make a release and add @koic on rubygems this weekend.

**Comment by mikegee** [comment]
I released 0.5.0.
